File: beginner/pagina03/1101/1101.lua
while true do
    local input = io.read("*l")
    
    local n, m = input:match("(%S+) (%S+)")
    n, m = tonumber(n), tonumber(m)
    
    if n <= 0 or m <= 0 then break end 
    
    local min = math.min(n, m)
    local max = math.max(n, m)

    local ans = ""
    local sum = 0
    for i=min, max do
        ans = ans .. string.format("%d", i) .. " "
        sum = sum + i
    end

    ans = ans .. "Sum=" .. string.format("%d", sum)
    print(ans)
end

File: beginner/__1000__/1101/1101.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    while (true) {
        int n, m;
        cin >> n >> m;
        if (n <= 0 || m <= 0) break;
        int i = min(n, m);
        int f = max(n, m);
        int sum = 0;
        string ans = "";
        for(size_t num=i; num<=f; num++){
            ans += to_string(num) + " ";
            sum += num;
        }
        ans += "Sum=" + to_string(sum) + "\n";
        cout << ans;
    }
    return 0;
}